fulp_modules/features/antagonists/bloodsuckers/code/monster_hunter/monsterhunter_datum.dm

```diff
@@ -37,44 +41,79 @@
 	//Give Monster Hunter powers
 	trackvamp.Grant(owner.current)
 	fortitude.Grant(owner.current)
+	//Give Hunter Objective
 	if(give_objectives)
-		//Give Hunter Objective
-		var/datum/objective/bloodsucker/monsterhunter/monsterhunter_objective = new
-		monsterhunter_objective.owner = owner
-		objectives += monsterhunter_objective
-		//Give Theft Objective
-		var/datum/objective/steal/steal_objective = new
-		steal_objective.owner = owner
-		steal_objective.find_target()
-		objectives += steal_objective
+		find_monster_targets()
+	var/datum/map_template/wonderland/wonder = new()
```

Comment:
You should add a check to not load several if there's multiple MHs.

Reply:
only 1 can spawn